aiida/backends/djsite/db/models.py

```diff
@@ -364,12 +355,15 @@ class DbComment(m.Model):
     content = m.TextField(blank=True)
 
     def __str__(self):
-        return 'DbComment for [{} {}] on {}'.format(self.dbnode.get_simple_name(),
-                                                    self.dbnode.pk, timezone.localtime(self.ctime).strftime('%Y-%m-%d'))
+        return 'DbComment for [{} {}] on {}'.format(
+            self.dbnode.get_simple_name(), self.dbnode.pk,
+            timezone.localtime(self.ctime).strftime('%Y-%m-%d')
+        )
 
 
 @python_2_unicode_compatible
```

Comment:
can you do a search for all these and just remove them?
I guess all of these decorators can be removed

Reply:
Ok

Reply:
done